modules/perception/camera/app/lane_camera_perception.cc

```diff
@@ -0,0 +1,239 @@
+    << "Failed to init " << lane_detector_plugin_param.name();
+    AINFO << "detector: " << lane_detector_->Name();
+
+    //  initialize lane postprocessor
+    auto lane_postprocessor_param =
+        perception_param_.lane_param().lane_postprocessor_param();
+    LanePostprocessorInitOptions postprocessor_init_options;
+    postprocessor_init_options.detect_config_root =
+        GetAbsolutePath(work_root, lane_detector_plugin_param.root_dir());
+    postprocessor_init_options.detect_config_name =
+        lane_detector_plugin_param.config_file();
+    postprocessor_init_options.root_dir =
+        GetAbsolutePath(work_root, lane_postprocessor_param.root_dir());
+    postprocessor_init_options.conf_file =
+        lane_postprocessor_param.config_file();
+    lane_postprocessor_.reset(
+        BaseLanePostprocessorRegisterer::GetInstanceByName(
+            lane_postprocessor_param.name()));
+    CHECK(lane_postprocessor_ != nullptr);
+    CHECK(lane_postprocessor_->Init(postprocessor_init_options))
+    << "Failed to init " << lane_postprocessor_param.name();
+    AINFO << "lane_postprocessor: " << lane_postprocessor_->Name();
+  }
+}
+
+void LaneCameraPerception::InitCalibrationService(
+            const std::string &work_root,
+            const base::BaseCameraModelPtr model,
+            const app::PerceptionParam &perception_param) {
+  // Init calibration service
+  CHECK(perception_param.has_calibration_service_param())
+  << "Failed to include calibration_service_param";
+  {
+    auto calibration_service_param =
+                      perception_param.calibration_service_param();
+    CalibrationServiceInitOptions calibration_service_init_options;
+    calibration_service_init_options.calibrator_working_sensor_name
+                    = lane_calibration_working_sensor_name_;
+    calibration_service_init_options.name_intrinsic_map = name_intrinsic_map_;
+    calibration_service_init_options.calibrator_method =
+                            calibration_service_param.calibrator_method();
+    calibration_service_init_options.image_height =
+      static_cast<int>(model->get_height());
+    calibration_service_init_options.image_width =
+      static_cast<int>(model->get_width());
+    calibration_service_.reset(
+          BaseCalibrationServiceRegisterer::GetInstanceByName(
+                            calibration_service_param.plugin_param().name()));
+    CHECK(calibration_service_ != nullptr);
+    CHECK(calibration_service_->Init(calibration_service_init_options))
+    << "Failed to init " << calibration_service_param.plugin_param().name();
+    AINFO << "calibration_service:: " << calibration_service_->Name();
+  }
+}
+
+void LaneCameraPerception::SetCameraHeightAndPitch(
+       const std::map<std::string, float> &name_camera_ground_height_map,
+       const std::map<std::string, float> &name_camera_pitch_angle_diff_map,
+       const float &pitch_angle_calibrator_working_sensor) {
```

Comment:
For primitive types, no need to use "&".

Reply:
done